go/oprf/oprf.go

```diff
@@ -73,7 +74,7 @@ func (ev Evaluation) ToJSON(verifiable bool) ([]byte, error) {
 type Participant interface {
 	Ciphersuite() gg.Ciphersuite
 	Setup(string, gg.PrimeOrderGroup) (Participant, error)
-	Blind([]byte) (gg.GroupElement, *big.Int, error)
+	Blind([]byte) (gg.GroupElement, gg.GroupElement, *big.Int, error)
```

Comment:
Do we want to be careful here? We're exposing an interface that is different to the interface exposed by the VOPRF draft. `BlindFixed` is used when generating test vectors, would there be a way of only modifying the interface for this function (which is not documented) and keeping the `Blind` interface the same?

Reply:
My take was that this only changes the implementation, and not the actual interface. Though on hindsight, you're probably right. Let me rework this a bit. Stand by!